telluric/collections.py

```diff
@@ -303,18 +303,6 @@ def rasterize(self, dest_resolution, *, polygonize_width=0, crs=WEB_MERCATOR_CRS
         else:
             return rasterize(shapes, crs, bounds.get_shape(crs), dest_resolution, fill_value=fill_value, dtype=dtype)
 
-    def plot(self, mp=None, max_plot_rows=200, **plot_kwargs):
```

Comment:
can you explain why you removing this method?

Reply:
Because it's already in the parent mixin, and it doesn't limit by rows. The logic is:

> With the previous situation, the user would get a warning
in both calls with the default parameters. Now, we
make the simple plot never freeze the browser, and
we remove the default limit in the plot() widget.

This is because `plot()` is decently fast, so we don't need the row limitation. And also, the user can always do `fc[:200].plot()`.